gmso/lib/jsons/PeriodicTorsionPotential.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "PeriodicTorsionPotential",
-  "expression": "k * (1 + cos(n * phi - phi_eq))",
+  "expression": "k0 + k1 * (1 + cos(1 * phi - phi_eq1)) + k2 * (1 + cos(2 * phi - phi_eq2)) + k3 * (1 + cos(3 * phi - phi_eq3)) + k4 * (1 + cos(4 * phi - phi_eq4)) + k5 * (1 + cos(5 * phi - phi_eq5))",
```

Comment:
After spending some more thought on this, we might want to be careful with this eqn, especially regarding how many terms we want to support. As in, can we keep the old untouched and create a different/derivative for 5 terms, 4 terms, etc. More ideas/thoughts are welcome

Reply:
Yeah, I hear what you mean.  I think, If you do not supported fixed terms it may be harder to parse when you don’t know what is there for the engines.  I believe We should be consistent, meaning if RB and OPLS is fixed this should be too.

Reply:
I also don’t believe I saw anything ever past n=5 or power=5,  which are both there converting to one or the other.  The fixed term way in my mind is consistent for programming and the readers.

Reply:
After additional though, if we decide to use the original periodic form, we should have additional check for the input variables. For example:

- The valuables must me k1, … kx, n1, …, nx or phi1, …, phix.  This will ensure the FFs are shareable.  

- k1 and phi1 must have n1=1 .  Kx and phix must have nx =x